lib/Doctrine/Migrations/Tools/Console/Command/MigrateCommand.php

```diff
@@ -59,7 +59,7 @@ protected function configure() : void
                 'allow-no-migration',
                 null,
                 InputOption::VALUE_NONE,
-                'Don\'t throw an exception if no migration is available (CI).'
+                'Do not throw an exception if no migration is available.'
```

Comment:
`Don't` definitely rings nicer, I don't think you should change that.
```suggestion
                'Don\'t throw an exception if no migration is available.'
```

Reply:
It was not my idea. Please see https://github.com/doctrine/migrations/pull/840#discussion_r307963742

Reply:
Oh, never mind then. In that case, just squash commits please :)

Reply:
done